config/initializers/twitter_regex.rb

```diff
@@ -26,7 +26,9 @@ class Regex
         )
       \)
     /iox
+    # rubocop:disable Layout/LineLength
     UCHARS = '\u{A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}\u{10000}-\u{1FFFD}\u{20000}-\u{2FFFD}\u{30000}-\u{3FFFD}\u{40000}-\u{4FFFD}\u{50000}-\u{5FFFD}\u{60000}-\u{6FFFD}\u{70000}-\u{7FFFD}\u{80000}-\u{8FFFD}\u{90000}-\u{9FFFD}\u{A0000}-\u{AFFFD}\u{B0000}-\u{BFFFD}\u{C0000}-\u{CFFFD}\u{D0000}-\u{DFFFD}\u{E1000}-\u{EFFFD}\u{E000}-\u{F8FF}\u{F0000}-\u{FFFFD}\u{100000}-\u{10FFFD}'
+    # rubocop:enable Layout/LineLength
```

Comment:
As a side note -- is our intention with this file to basically "vendor" portions of this? https://github.com/twitter/twitter-text/blob/master/rb/lib/twitter-text/regex.rb

If so, some of it has changed a bit from what we have here.